spec/integration.rb

```diff
@@ -0,0 +1,138 @@
+# frozen_string_literal: true
+
+# ------------------------------------- #
+#  cocoapods-bazel Integration tests    #
+# ------------------------------------- #
+
+#-----------------------------------------------------------------------------#
+
+# The following integrations tests are based on file comparison.
+#
+# 1.  For each test there is a folder with a `before` and `after` subfolders.
+# 2.  The contents of the before folder are copied to the `TMP_DIR` folder and
+#     then the given arguments are passed to the `POD_BINARY`.
+# 3.  After the pod command completes the execution the each file in the
+#     `after` subfolder is compared to be to the contents of the temporary
+#     directory.  If the contents of the file do not match an error is
+#     registered. Xcode projects are compared in an UUID agnostic way.
+#
+# Notes:
+#
+# - The output of the pod command is saved in the `execution_output.txt` file
```

Comment:
Is this valid? I don't see any `execution_output.txt` files. Lifted from CP integration tests?

Reply:
If not, maybe some of the `s.replace_pattern` below aren't necessary either? Fine to keep if you think it would be useful in the future.

Reply:
yeah, copy-pasted from cocoapods-generate which in turn was copy-pasted from cocoapods. will keep the output filtering in case we ever do output more than a single line from the plugin